index.js

```diff
@@ -1,5 +1,7 @@
 // transpile:main
 
 import ADB from './lib/adb';
+import { DEFAULT_ADB_PORT } from './lib/adb';
 
 export default ADB;
+export { DEFAULT_ADB_PORT };
```

Comment:
Cool!

I like the pattern, however, of putting the default into the named exports as well. That way if you just want the default you can use

```js
import ADB from 'appium-adb';
```

but if you want the default and something else you can use

```js
import { ADB, DEFAULT_ADB_PORT } from 'appium-adb';
```

and not 

```js
import { default as ADB, DEFAULT_ADB_PORT } from 'appium-adb';
```